internal/incoming/odfi/processor.go

```diff
@@ -150,9 +152,12 @@ func processFile(path string, auditSaver *AuditSaver, fileProcessors Processors)
 	dir, filename := filepath.Split(path)
 	dir = filepath.Base(dir)
 
+	// Allow an override of root path or use legacy path
+	folder := strx.Or(os.Getenv("RDFI_FOLDER"), "odfi")
+
 	// Persist the file if needed
 	if auditSaver != nil {
-		path := fmt.Sprintf("odfi/%s/%s/%s/%s", auditSaver.hostname, dir, time.Now().Format("2006-01-02"), filename)
+		path := fmt.Sprintf("%s/%s/%s/%s/%s", folder, auditSaver.hostname, time.Now().Format("2006-01-02"), dir, filename)
```

Comment:
This breaks ach-web-viewer as it is today, so I'm hesitant to merge this change. I know it's more convenient but we have precedent already.